### PR TITLE
Add Image Process

### PR DESCRIPTION
**Pull request by zhaohuxing**

### CODE REVIEW

File: helpers/image.go [hunk 1]
```diff
@@ -0,0 +1,265 @@
+package helpers
+
+import (
+	"fmt"
+	"github.com/yunify/qingstor-sdk-go/service"
+	"reflect"
+	"strings"
+)
+
```
Comment (pengsrc) [inline helpers/image.go]: Import order is not tidy.

``` Go
import (
	"fmt"
	"reflect"
	"strings"

	"github.com/yunify/qingstor-sdk-go/service"
)
```

File: helpers/image_test.go [hunk 2]
```diff
@@ -0,0 +1,61 @@
+package helpers
+
+import (
+	"github.com/stretchr/testify/assert"
+	qs "github.com/yunify/qingstor-sdk-go/service"
+	"testing"
+)
+
```
Comment (pengsrc) [inline helpers/image_test.go]: Import order is not tidy.

File: test/main.go [hunk 3]
```diff
@@ -58,7 +59,6 @@ func setUp() {
 	loadTestConfig()
 	loadConfig()
 	initQingStorService()
-
```
Comment (pengsrc) [inline test/main.go]: Why?

**Review by pengsrc — CHANGES_REQUESTED** [review]

File: client/image.go [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+	// CropAuto crops image to auto width and height.
+	CropAuto
+)
+
+// Image is the combination type usually image process
+type Image struct {
+	ImageOutput *service.ImageProcessOutput
+	bucket      *service.Bucket
+	uri         *string
+	name        *string
```
Comment (Xuanwo) [inline client/image.go]: Use `objectKey` directly here.

**Review by Xuanwo — CHANGES_REQUESTED** [review]

File: client/image.go (hunk 4 above)
Comment (zhaohuxing) [reply client/image.go]: OK, Thanks.

**Comment by Xuanwo** [comment]
Rebase your code from master.

**Comment by zhaohuXing** [comment]
Done.

**Review by pengsrc — APPROVED** [review]

**Review by Xuanwo — APPROVED** [review]

**Pull request merged**